### PR TITLE
Install the Forest middleware and declare a simple 'Merge' action on …

### PR DESCRIPTION
**Pull request by SeyZ**
…User collection

### CODE REVIEW

File: server/lib/express.js [hunk 1]
```diff
@@ -74,6 +74,14 @@ export default function(app) {
 
   app.use(multer());
 
+  app.use(require('forest-express-sequelize').init({
```
Comment (asood123) [inline server/lib/express.js]: Our linter won't be happy with this. Mind moving this to the top as an Import statement?

File: server/forest/User.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const Liana = require('forest-express-sequelize');
+
+Liana.collection('User', {
```
Comment (asood123) [inline server/forest/User.js]: Can you explain what this will do?

File: package.json [hunk 3]
```diff
@@ -42,6 +42,7 @@
     "express-server-status": "1.0.3",
     "express-session": "1.14.0",
     "flat": "2.0.0",
+    "forest-express-sequelize": "1.5.4",
```
Comment (asood123) [inline package.json]: Thanks for removing the carat :).

File: server/forest/User.js (hunk 2 above)
Comment (SeyZ) [reply server/forest/User.js]: https://doc.forestadmin.com/developer-guide/lumber.html#what-is-a-smart-action

**Comment by xdamman** [comment]
Guys, I'm in Brussels. What about we do some pair programming to get this out of the door? Let me know what works.

**Comment by SeyZ** [comment]
Hey @xdamman I won't be in Brussels this week. I can improve the linter things. Then, I would be happy to help if needed. But everything should be straighforward. Let me know

**Comment by SeyZ** [comment]
Job done on my side. I'll let you merge and test your new admin ;-) Let me know if you need anything else.

**Comment by xdamman** [comment]
Where is the logic for merging two users?

**Comment by SeyZ** [comment]
Hey @xdamman - I've bootstraped the Forest installation but I don't know exactly the business logic needed to merge two users. It's related to your business, not to Forest. I guess you will be much more productive than me on this right ? :-)

**Comment by xdamman** [comment]
Sure but can you put a placeholder / example? Always easier that way.

**Comment by znarf** [comment]
Closing, merged a similar PR: https://github.com/opencollective/opencollective-api/pull/1242

**Pull request closed without merging**